Installation/cmake/modules/CGAL_add_test.cmake

```
@@ -74,13 +77,26 @@ function(expand_list_with_globbing list_name)
 endfunction()
 
 function(cgal_add_compilation_test exe_name)
-  if(TEST compilation_of__${exe_name})
-    return()
-  endif()
-  add_test(NAME "compilation_of__${exe_name}"
-    COMMAND ${TIME_COMMAND} "${CMAKE_COMMAND}" --build "${CMAKE_BINARY_DIR}" --target "${exe_name}" --config "$<CONFIG>")
-  set_property(TEST "compilation_of__${exe_name}"
-    APPEND PROPERTY LABELS "${PROJECT_NAME}")
+  if(NOT CGAL_WIN32_CMAKE_ON_CYGWIN)
```

I suggest to test the variable [`CMAKE_VS_MSBUILD_COMMAND`](https://cmake.org/cmake/help/latest/variable/CMAKE_VS_MSBUILD_COMMAND.html) directly:
```suggestion
  if(NOT CMAKE_VS_MSBUILD_COMMAND)
```